### PR TITLE
Add LayerTransparencySlider

### PR DESCRIPTION
**Pull request by dnlkoch**
This introduces the `LayerTransparencySlider`.

### CODE REVIEW

File: src/Slider/LayerTransparencySlider.example.jsx [hunk 1]
```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { render } from 'react-dom';
+
+import OlMap from 'ol/map';
+import OlView from 'ol/view';
+import OlTileLayer from 'ol/layer/tile';
+import OlOsmSource from 'ol/source/osm';
```
Comment (KaiVolland) [inline src/Slider/LayerTransparencySlider.example.jsx]: Plz use import names as declared in #58 
so `OlTileLayer` would become `OlLayerTile`

Comment (dnlkoch) [reply src/Slider/LayerTransparencySlider.example.jsx]: Thanks for the hint!

File: src/Slider/LayerTransparencySlider.jsx [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+   * Returns the transparency from the provided layer.
+   *
+   * @return {Number} The transparency of the layer.
+   */
+  getLayerTransparency() {
+    // 1 = fully opaque/visible.
+    let opacity = this.props.layer.getOpacity();
+    let transparency = (1 - opacity) * 100;
+    // Remove any digits.
+    transparency = Math.round(transparency);
```
Comment (marcjansen) [inline src/Slider/LayerTransparencySlider.jsx]: Why? are you sure this is the right thing to do?

File: src/Slider/LayerTransparencySlider.jsx [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+      ...passThroughProps
+    } = this.props;
+
+    return (
+      <Slider
+        tipFormatter={value => `${value}%`}
+        defaultValue={this.getLayerTransparency()}
+        onChange={(value) => {
+          this.setLayerTransparency(value);
+        }}
```
Comment (marcjansen) [inline src/Slider/LayerTransparencySlider.jsx]: Should we use `this.setLayerTransparency.bind(this)`, instead of the arrow function?

**Pull request merged**